cucumber/kubernetes/features/step_definitions/authenticate_steps.rb

```diff
@@ -56,10 +56,14 @@ def authenticate_k8s(host, cert, key, conjur_id)
   end
 end
 
-Then(/^I( can)? authenticate pod matching "([^"]*)" with authn-k8s as "([^"]*)"( without cert and key)?$/) do |success, objectid, hostid, nocertkey|
+Then(/^I( can)? authenticate pod matching "([^"]*)" with authn-k8s as "([^"]*)"( with prefix "([^"]*)")?( without cert and key)?$/) do |success, objectid, hostid_suffix, has_custom_prefix, hostid_prefix, nocertkey|
```

Comment:
What does `"([^"]*)")` actually mean here? Is that regex?

Reply:
yes